fury/window.py

```diff
@@ -399,11 +399,58 @@ def __init__(self, scene=None, title='FURY', size=(300, 300),
         self.style.SetInteractor(self.iren)
         self.iren.SetInteractorStyle(self.style)
         self.iren.SetRenderWindow(self.window)
+        self._timelines = []
+        self._timeline_callback = None
 
     def initialize(self):
         """Initialize interaction."""
         self.iren.Initialize()
 
+    @property
+    def timelines(self):
+        return self._timelines
+
+    def add_timeline(self, timeline: Timeline):
+        """Add a Timeline to the ShowManager.
+        Adding a Timeline to the ShowManager ensures that it gets added to
+        the scene, gets updated and rendered without any extra code.
+
+        Parameters
+        ----------
+        timeline : Timeline
+            The Timeline to be added to the ShowManager.
+        """
+
+        self.scene.add(timeline)
+        if timeline in self._timelines:
+            return
```

Comment:
strange that this condition arrive after `self.scene.add(timeline)`. I would recommend to move it before to avoid to  add the timeline twice in the scene

Reply:
You are right